phpBB/includes/functions_compatibility.php

```diff
@@ -943,3 +943,76 @@ function parse_cfg_file($filename, $lines = false)
 
 	return $parsed_items;
 }
+
+/**
+* Wraps an url into a simple html page. Used to display attachments in IE.
+* this is a workaround for now; might be moved to template system later
+* direct any complaints to 1 Microsoft Way, Redmond
+*
+* @deprecated: 3.3.0-dev (To be removed: 4.0.0)
+*/
+function wrap_img_in_html($src, $title)
+{
+	echo '<!DOCTYPE html>';
+	echo '<html>';
+	echo '<head>';
+	echo '<meta charset="utf-8">';
+	echo '<meta http-equiv="X-UA-Compatible" content="IE=edge">';
+	echo '<title>' . $title . '</title>';
+	echo '</head>';
+	echo '<body>';
+	echo '<div>';
+	echo '<img src="' . $src . '" alt="' . $title . '" />';
+	echo '</div>';
+	echo '</body>';
+	echo '</html>';
+}
+
+/**
+* Garbage Collection
+*
+* @param bool $exit		Whether to die or not.
+*
+* @return null
+*
+* @deprecated: 3.3.0-dev (To be removed: 4.0.0)
+*/
+function file_gc($exit = true)
+{
+	global $cache, $db;
+
+	if (!empty($cache))
+	{
+		$cache->unload();
+	}
+
+	$db->sql_close();
+
+	if ($exit)
+	{
+		exit;
+	}
+}
+
+/**
+* Check if the browser is internet explorer version 7+
+*
+* @param string $user_agent	User agent HTTP header
+* @param int $version IE version to check against
+*
+* @return bool true if internet explorer version is greater than $version
+*
+* @deprecated: 3.3.0-dev (To be removed: 4.0.0)
+*/
+function phpbb_is_greater_ie_version($user_agent, $version)
```

Comment:
Same here, IE support should be removed.

Reply:
Yes, that is why it is in functions_compatibility

Reply:
I think functions_compatibility should only be used for functions that might still be used, e.g. by extensions. Since we're moving for 4.0, I think it's perfectly fine to break some ties and remove functions that IMHO shouldn't be used outside core phpBB and will not make any sense to use anymore.
However, I'm actually wondering how we can best communicate these removed functions. I'm currently thinking about a dedicated page in the development documentation?